### PR TITLE
OSDOCS-6344-RN: updates GA for InFW Operator

### PR DESCRIPTION
**Pull request by JoeAldinger**
OSDOCS#6344-RN: updates GA for InFW Operator

Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-6344

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:
https://github.com/openshift/openshift-docs/pull/60740

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -72,6 +72,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-14-networking"]
 === Networking
 
+[id="ocp-4-14-ingress-node-firewall-operator-ga"]
+== Ingress Node Firewall Operator (General Availability)
+Ingress Node Firewall Operator was made a technology preview in {product-title} 4.12. With this release, Ingress Node Firewall Operator is generally available. You can now configure firewall rules at the node level. For more information, see xref:../networking/networking-operators-overview.adoc#networking-operators-overview[Ingress Node Firewall Operator].
+
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:78]: Looking at the 4.13 release notes, it seems most GA announcements contain _is generally available_ in the heading:
Ingress Node Firewall Operator  is generally available

Comment (JoeAldinger) [reply release_notes/ocp-4-14-release-notes.adoc:78]: They sure do! 😄

**Pull request merged**